src/entry-server.jsx

```diff
@@ -3,8 +3,7 @@ import { DataProvider } from "./data";
 import { ABORT_DELAY, API_DELAY } from "./delays";
 import {
   renderToPipeableStream,
-  renderToReadableStream,
-} from "react-dom/server";
+} from "react-dom/cjs/react-dom-server.node.development";
```

Comment:
It looks like this is using node specific code.  Actually, that looks like that was already the case with renderToPipeableStream.

Reply:
I had to provide a very specific path because we set the `target` to `webworker`, and so yarn imports browser-specific stuff but we need the node implementation.

As for the node-specific code, it only uses `setImmediate` and `Buffer.from` so far, we have `Buffer` and `setImmediate` is easy to polyfill (or can be implemented natively in Oxygen). The `renderToReadableStream` implementation didn't use any specific node stuff but it sits in `react-dom-server.browser.development.js` which suggests it might be eventually using browser-specific APIs. React doesn't target workers as a platform yet, but if we get React streaming to production quality in Oxygen, this can change.